### PR TITLE
chore(GroupHelper): minor tweak

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
refactoring

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
Beautify code.

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: lib/utils/group-helper.js [hunk 1]
```diff
@@ -57,8 +57,7 @@ class GroupHelper {
     }
     mapArgToBoolean(name, optionName) {
         ifArg(name, function(bool) {
-            if (bool === true) options[optionName || name] = true;
-            else if (bool === false) options[optionName || name] = false;
+            options[optionName || name] = bool;
```
Comment (ematipico) [inline lib/utils/group-helper.js]: What if `bool` is not a real boolean?

Comment (jamesgeorge007) [reply lib/utils/group-helper.js]: The check assumes `bool` to be either `true` or `false` right?

Comment (jamesgeorge007) [reply lib/utils/group-helper.js]: Oh yeah, making sure none other than `true` or `false` comes up right?

Comment (ematipico) [reply lib/utils/group-helper.js]: Yes exactly. Your change changes the functionality of the code.

**Pull request merged**